14_functions_lab/01.py

```diff
@@ -1,7 +1,23 @@
-"""
-Write 2 functions:
-    mysum - returns the sum of its input arguments
-    mymul - returns the multiplication of its input arguments
-    Ignore non-numeric arguments
-"""
-
+"""
+      _      _      _
+   __(.)< __(.)> __(.)=
+   \___)  \___)  \___)   
+
+Python course - Verint
+Week 2
+
+"""
+
+def mysum(*arg):
+    res = 0

```

Comment:
You can also use Python's `sum()` function, by replacing the loop with list comprehension